test/test.c

```diff
@@ -2390,6 +2391,10 @@ test_natsOptions(void)
         s = natsOptions_SetMaxPingsOut(opts, 10);
     testCond((s == NATS_OK) && (opts->maxPingsOut == 10));
 
+    test("Set IOBufSize: ");
+    s = natsOptions_SetIOBufSize(opts, 1024 * 1024);
+    testCond((s == NATS_OK) && (opts->ioBufSize == 1024 * 1024));
+
```

Comment:
Here, you could have a test that checks negative value (and expect s == NATS_INVAID_ARG) and then set to 0 and expect the size to be set to NATS_OPTS_DEFAULT_IO_BUF_SIZE.

Reply:
See d086d8d.